src/components/QuickLinksDialog/index.tsx

```diff
@@ -0,0 +1,98 @@
+import GordonDialogBox from 'components/GordonDialogBox';
+import LinkIcon from '@mui/icons-material/InsertLink';
+import { ListItemIcon, ListItemText, ListSubheader, List, ListItem } from '@mui/material';
+import CanvasIcon from './images/Canvas.ico';
+import GordonIcon from './images/favicon.ico';
+import GOIcon from './images/GoGordonFavicon.ico';
+import MyGordonIcon from './images/MyGordonFavicon.ico';
+import { PropsWithChildren } from 'react';
+
+type Props = {
+  handleLinkClose: () => void;
+  linkopen: boolean;
+};
+
+const customIconLinks: ListItemProps[] = [
+  {
+    href: 'https://www.gordon.edu',
+    name: 'Gordon College',
+    icon: {
+      src: GordonIcon,
+      alt: 'gordon',
+    },
+  },
+  {
+    href: 'https://my.gordon.edu',
+    name: 'My Gordon',
+    icon: { src: MyGordonIcon, alt: 'my gordon' },
+  },
+  { href: 'https://go.gordon.edu', name: 'Go Gordon', icon: { src: GOIcon, alt: 'go gordon' } },
+  { href: 'https://canvas.gordon.edu', name: 'Canvas', icon: { src: CanvasIcon, alt: 'canvas' } },
+];
+
+const otherLinks = [
+  {
+    href: 'http://athletics.gordon.edu/',
```

Comment:
```suggestion
    href: 'https://athletics.gordon.edu/',
```
Consider me triggered